packages/core/src/SlideBox/SlideBoxWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes, { InferProps } from 'prop-types'
+import { Box } from '../Box'
+import { Relative } from '../Relative'
+import { Flex } from '../Flex'
+
+const propTypes = {
+  children: PropTypes.node,
+  arrowPosition: PropTypes.string,
```

Comment:
n-b changing `propTypes` to an interface and making `arrowPosition` a union of possible values would be nice

Reply:
Updated